src/backend/joanie/tests/swagger/swagger.json

```diff
@@ -5879,6 +6008,17 @@
                     "title"
                 ]
             },
+            "State693Enum": {
```

Comment:
We can add `"OrderStateEnum": "joanie.core.enums.ORDER_STATE_CHOICES",` to https://github.com/openfun/joanie/blob/main/src/backend/joanie/settings.py#L297 to avoid this generated name.
Please note that it will also update `src/backend/joanie/tests/swagger/admin-swagger.json` but in a good way (IMHO)